### PR TITLE
Pl 92 authentication

### PR DESCRIPTION
**Pull request by KBalderson**
Registration and Authentication working. Omniauth also working.

### CODE REVIEW

File: app/controllers/private_label/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
+class PrivateLabel::UsersController < PrivateLabel::plController
```
Comment (dmkash) [inline app/controllers/private_label/users_controller.rb]: Should this be renamed PeopleController?

**Comment by KBalderson** [comment]
@dmkash I think it's just an artifact from earlier commits... didn't you already move it once?

**Comment by KBalderson** [comment]
oh, maybe not. I'll rename it

**Comment by dmkash** [comment]
@KBalderson I haven't moved anything outside of the admin controllers, no :(  Otherwise looks great!

**Comment by KBalderson** [comment]
@dmkash I renamed it... and made other small namespace-y changes

**Comment by dmkash** [comment]
:+1:

**Pull request merged**